src/fetchconfig/cache.js

```diff
@@ -71,7 +71,7 @@ function cache(fn, opts = {}) {
  * Resets the QuickLRU cache with new options. Existing cache entries
  * will be cleared.
  * @param {object} opts options
- * @param {integer} opts.maxSize maximum size of the cache
+ * @param {number} opts.maxSize maximum size of the cache
```

Comment:
Is it a number? What would a cache with 1.5 entries look like?

Reply:
well, jsdoc mandates to use the js types. `integer` is not one.